### DISCUSSION TITLE
Unable to deposit

### DISCUSSION THREAD

**Discussion by TanyaSrivastava-Dipali** [bcnmy/biconomy-client-sdk#462] [Q&A] [answered]
I've connect metamask wallet for polygon  mumbai network.When I'm clicking on deposit button.It doesn't open metamask popup or anything.Just getting this error .
Access to fetch at 'https://rpc-mumbai.maticvigil.com/' from origin 'https://dashboard.biconomy.io' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

how it can be fixed

**Comment by himanshugarg06** [comment, marked as answer]
Mumbai testnet will be [deprecated](https://www.alchemy.com/blog/polygon-mumbai-testnet-deprecation) soon, thats why the RPC issues.  It's recommended to transition to the Amoy testnet. The Biconomy infra now supports amoy testnet.
